docs/canvas/canvas-share-workpad.asciidoc

```diff
@@ -2,47 +2,94 @@
 [[workpad-share-options]]
 == Share your workpad
 
-When you are ready to share your workpad, create a PDF, or export your workpad.
+When you've finished your workpad, you can share it outside of {kib}.
+
+[float]
+[[export-single-workpad]]
+=== Export workpads
+
+Create a JSON file of your workpad that you can export outside of {kib}.
+
+. From your workpad, click the *Share workpad* icon in the upper left corner.
+
+. Select *Download as JSON*.
++
+[role="screenshot"]
+image::images/canvas-export-workpad.png[Export single workpad]
+
+Want to export multiple workpads? Go to the *Canvas workpads* page, select the workpads you want to export, then click *Export*.
```

Comment:
page or view? I typically use view

export, then click > export, **and** then 

If you make the change above, you should probably do it throughout the doc.

Reply:
The less words, the better :) I always leave out `then`s and `and then`s. They aren't technically necessary to read the sentence.